warehouse/query-core/src/main/java/datawave/query/attributes/Document.java

```diff
@@ -211,17 +211,8 @@ public Attribute<?> toDocKeyAttributes(Set<Key> docKeys, boolean keepRecordId) {
 
     public void debugDocumentSize(Key docKey) {
         long bytes = sizeInBytes();
-        // if more than 100M, then error
-        if (bytes > (ONE_HUNDRED_M)) {
-            log.error("Document " + docKey + "; size = " + size() + "; bytes = " + bytes);
-        }
-        // if more than 10M, then warn
-        // else if (bytes > (1024l * 1000 * 10)) {
-        // log.warn("Document " + docKey + "; size = " + size() + "; bytes = " + bytes);
-        // }
-
         // if more than 1M, then info
-        else if (bytes > (ONE_M)) {
+        if (bytes > (ONE_M)) {
```

Comment:
Perhaps too much info now.  Make this ONE_HUNDRED_M and the debug below for ONE_M.